### PR TITLE
RDM-9893 - Remove OrganisationName references

### PR DESCRIPTION
**Pull request by danlysiak**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-9893

### Change description ###
Remove references from OrganisationName

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/aac-manage-case-assignment/values.preview.template.yaml [hunk 1]
```diff
@@ -12,4 +12,4 @@ java:
         - notify-mca-api-key-secret
   environment:
     S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-    CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-pr-601.service.core-compute-preview.internal
+    CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-pr-1187.service.core-compute-preview.internal
```
Comment (sullivangareth) [inline charts/aac-manage-case-assignment/values.preview.template.yaml:15]: Are these changes related or should they be in a separate review ?

File: build.gradle [hunk 2]
```diff
@@ -282,7 +282,7 @@ dependencies {
   integrationTestImplementation sourceSets.main.runtimeClasspath
   integrationTestImplementation sourceSets.test.runtimeClasspath
 
-  testImplementation group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.9.0'
+  testImplementation group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.19.0-RDM-9893-3'
```
Comment (sullivangareth) [inline build.gradle:285]: Are these changes related or should they be in a separate review ?

**Review by sullivangareth — APPROVED** [review]

File: charts/aac-manage-case-assignment/values.preview.template.yaml (hunk 1 above)
Comment (danlysiak) [reply charts/aac-manage-case-assignment/values.preview.template.yaml:15]: They'll be reverted before merging 👍 Just temporarily pointing to the PRs where I've made similar changes to ensure it all works e2e.

File: build.gradle (hunk 2 above)
Comment (danlysiak) [reply build.gradle:285]: There were updates to the spreadsheets required as part of the changes, so created a pre-release version - will be set appropriately with a complete release as part of the standard QA process 👍
